source/testing/testing-components.md

```diff
@@ -128,183 +110,70 @@ export default Ember.Component.extend({
 jQuery triggers can be used to simulate user interaction and test that the title
 is updated when the button is clicked on:
 
-```tests/unit/components/magic-title-test.js
-test('clicking the button updates the title', function(assert) {
+```tests/integration/components/magic-title/component-test.js
+test('should update title on button click', function (assert) {
   assert.expect(2);
 
-  // Create the component instance
-  var component = this.subject();
+  this.render(hbs`{{magic-title}}`);
 
-  // Assert the initial title
-  var initialTitle = this.$().find('h2').text();
-  assert.equal(initialTitle, 'Hello World');
+  assert.equal(this.$('h2').text(), 'Hello World', 'initial text is hello world');
 
-  // Click on the button
-  this.$().find('button').click();
+  //Click on the button
+  this.$('button').click();
 
-  // Assert that the title has changed
-  var finalTitle = this.$().find('h2').text();
-  assert.equal(finalTitle, 'Hello Ember World');
+  assert.equal(this.$('h2').text(), 'This is Magic', 'title changes after click');
 });
 ```
 
 ### Testing Actions
 
-Components often utilize the `sendAction()` method to send actions to other
-objects in your application.
+Components starting in Ember 2 utilize closure actions, which allow the component to directly invoke functions provided by components higher up in then tree.
```

Comment:
Can we simplify this to something like "If a component is nested within another component, the inner component can directly invoke functions provided by the outer component."?